src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
```

Comment:
```suggestion
Okteto works in air gapped network environments, but will require some custom configuration.
```

Reply:
I think we should remove this line entirely. Anyone doing an airgapped install knows that extra steps are required.  Could we instead talk about what we mean by air gap? (e.g. is a good example IMO https://docs.gitlab.com/ee/topics/offline/quick_start_guide.html)